### PR TITLE
P1

### PR DESCRIPTION
**Pull request by pritam-kr**
- [Created Simple profile page](https://github.com/pritam-kr/playme/commit/8aeebafb7bdfb46eb7025e14acef62bb1a8380d6)
- [Created custom hooks for notes](https://github.com/pritam-kr/playme/commit/3a98aec8129a89df39773755910b094c6cbed7c9)[](https://github.com/pritam-kr)
- [Working on notes app](https://github.com/pritam-kr/playme/commit/10f1b02d474674add6d810fad1c14c690f43cb22) Not completed Yet

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
 import "./App.css";
 import { Home, SortByLatest } from "./Pages/Index";
 import { Routes, Route, useLocation, Navigate } from "react-router-dom";
-import { Explore,Login, Playlist, SingleVideo,Signup, WatchLater , Likes, History,PlaylistVideos} from "./Pages/Index";
+import { Explore, Login, Playlist, SingleVideo,Signup, WatchLater , Likes, History,PlaylistVideos,Profile} from "./Pages/Index";
```
Comment (varnita619) [inline src/App.js:4]: good use of barrel exports

**Review by varnita619 — APPROVED** [review]
code looks clean

File: src/App.js [hunk 2]
```diff
@@ -18,8 +20,13 @@ function App() {
   }
  
 const PrivateRoute = ({children}) =>{
+
+  console.log(isAuth)
```
Comment (Sen-442b) [inline src/App.js]: You can remove redundant debuggers

**Review by Sen-442b — APPROVED** [review]
- Code  and file structure is organised
- Consistent Variable  naming

File: src/App.js (hunk 2 above)
Comment (pritam-kr) [reply src/App.js]: Done

**Pull request merged**